src/components/Alert.tsx

```diff
@@ -0,0 +1,18 @@
+import { ReactNode } from "react";
+import { IconButton } from "src/components/IconButton";
+import { Css, Palette } from "src/Css";
+
+export interface AlertProps {
+  children: ReactNode;
+  onClose: () => void;
+}
+
+export function Alert(props: AlertProps) {
```

Comment:
**Question:** Is there going to be a Context and Portal approach made for this alert component to make it easier to use similar to the Modal and SuperDrawer?

Reply:
@KoltonG good question; I didn't really have anything in mind other than the look & feel. You're thinking of basically a toast-ish sort of thing where alerts are auto-floated on top of the current context?

I like that; as-is pages would just have to layout the alert inline where ever they wanted.